application/gui/properties_window.py

```diff
@@ -193,6 +197,8 @@ def _rename_item(self, widget=None, data=None):
 
 	def _create_monitor(self):
 		"""Create item monitor"""
+		if hasattr(self, '_monitor') and self._monitor is not None and not self._monitor.is_cancelled():
+			self._monitor.cancel()
```

Comment:
I am noticing that trend about pull requests. Can't say am sorry for that to be the case as I should be able to join in on development soon. As for this change, am not sure it's a good idea to `_monitor` like that. I'll have to look deeper into that so I'll wait a bit with merge.